rita/src/rita_common/dashboard/network_endpoints.rs

```diff
@@ -38,3 +43,58 @@ pub fn set_settings(
 
     JsonStatusResponse::new(Ok("New settings applied".to_string()))
 }
+
+pub fn wipe(_req: HttpRequest) -> Result<HttpResponse, Error> {
+    if cfg!(not(debug_assertions)) {
+        return Ok(HttpResponse::NotFound().finish());
+    }
+
+    // Clean up existing WG interfaces
+    match cleanup() {
+        Ok(_) => trace!("Cleanup success!"),
+        Err(e) => {
+            warn!("Unable to complete cleanup: {:?}", e);
+            return Err(e);
```

Comment:
Cleanups shouldn't interrupt on errors but just move on with other stuff on their agenda. You should want to discard this error.

Reply:
maybe you want a stack trace?

Reply:
@jkilpatr A way to do both is what we need IMHO. However @mpapierski don't Rust errors stringify along with the backtrace?

Reply:
I'm not convinced on ignoring errors, though. I'd rather return error 500 at any point that would indicate that you'd have to do some action manually, or perhaps retry a HTTP request later from client side.

Reply:
@mpapierski Requiring developers to do stuff manually on a failed step of cleanup defeats the purpose of said cleanup or even purpose as a development enhancement. I say we return a JSON like:
```json
{
"wireguard_cleanup": "OK",
"restore_default_route": "OK",
"new_wg_key": "Failed",
"wg_key_write_to_disk": "Failed",
"new_mesh_ip": "OK",
}
```